Update orientation-epic.md

CherylEvans opened the pull request:
Line 45 - Added a new task under General to watch the Platform Support overview video
Line - 46 Added a new task under General to review the Getting help from the Platform in Slack guidance

Ticket 61713 for reference

CherylEvans committed:
Update orientation-epic.md

Line 45 - Added a new task under General to watch the Platform Support overview video
 Line - 46 Added a new task under General to review the Getting help from the Platform in Slack guidance

Ticket 61713 for reference

CherylEvans committed:
Update orientation-epic.md

Removed the word "Watch."

shiragoodman commented on .github/ISSUE_TEMPLATE/orientation-epic.md:
```diff
@@ -41,7 +41,9 @@ As a new VFS team member, I need to complete onboarding activities so I can succ
 - [ ] Watch [VA.gov overview video](https://depo-platform-documentation.scrollhelp.site/getting-started/Platform-Orientation.1877344532.html#PlatformOrientation-Purpose)
 - [ ] Attend the [General Orientation session](https://depo-platform-documentation.scrollhelp.site/getting-started/calendly-orientation-sessions#OrientationSessions-Generalorientation) (schedule via Calendly)
```
Please delete this line (42) as it is a duplicate of line 40

shiragoodman commented on .github/ISSUE_TEMPLATE/orientation-epic.md:
```diff
@@ -41,7 +41,9 @@ As a new VFS team member, I need to complete onboarding activities so I can succ
 - [ ] Watch [VA.gov overview video](https://depo-platform-documentation.scrollhelp.site/getting-started/Platform-Orientation.1877344532.html#PlatformOrientation-Purpose)
 - [ ] Attend the [General Orientation session](https://depo-platform-documentation.scrollhelp.site/getting-started/calendly-orientation-sessions#OrientationSessions-Generalorientation) (schedule via Calendly)
 - [ ] Watch [VA.gov overview video](https://depo-platform-documentation.scrollhelp.site/getting-started/Platform-Orientation.1877344532.html#PlatformOrientation-Purpose) for an overview of VA.gov
```
Please delete this line (43) as it is a duplicate of line 41.

CherylEvans committed:
Update orientation-epic.md

Removed Line 42 as a duplicate to line 40

CherylEvans committed:
Update orientation-epic.md

Deleted: Watch [VA.gov overview video](https://depo-platform-documentation.scrollhelp.site/getting-started/Platform-Orientation.1877344532.html#PlatformOrientation-Purpose) for an overview of VA.gov (Duplicate)
